app/shared/hooks/useIssues.js

```diff
@@ -2,4 +2,17 @@
 import { useContext } from 'react'
 import { IssuesContext } from './IssuesProvider'
 
-export const useIssues = () => useContext(IssuesContext)
+export const useIssues = () => {
+  const context = useContext(IssuesContext)
+  if (!context) {
+    throw new Error('useIssues must be used within an IssuesProvider')
+  }
+
+  const [issues, setIssues, refreshIssues] = context
+
+  return {
+    issues,
+    setIssues,
+    refreshIssues,
+  }
```

Comment:
Not sure why we want to convert this signature from `Array` -> `Object`? 🤔 

```suggestion
  return context
```